### DISCUSSION TITLE
kairos.io DNS_PROBE_FINISHED_NXDOMAIN

### DISCUSSION THREAD

**Discussion by x-0D** [kairos-io/kairos#1596] [General]
kairos.io domain is dead today.
https://digwebinterface.com/?hostnames=kairos.io&ns=all

**Comment by saamalik** [comment]
Apologies. The domain renewal had lapsed. It's already renewed, but still waiting for the registrant to point to proper NS records. 

Thanks for reporting!

**Comment by saamalik** [comment]
Ok, seems that most DNS resolvers have updated: https://digwebinterface.com/?hostnames=kairos.io&type=&useresolver=8.8.4.4&ns=all&nameservers=.

**Comment by Itxaka** [comment]
Thanks @x-0D for reporting and @saamalik for fixing it!
